cves/CVE-2014-8547.yml

```diff
@@ -257,10 +256,8 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: '3fd60d804996031ceaba9cad0b38652b92551eb0'
+    note: 'This commit ports the GIF decoder to use a new API that involves changing code that reads the colors and dimensions of the GIF, but does not touch the code that caused the vulnerability.'
```

Comment:
Personally, I am not a fan of long strings sitting on one line. I think utilizing a | here could increase the readability of this section of the document.